src/android/CameraLauncher.java

```diff
@@ -163,7 +159,19 @@ else if ((srcType == PHOTOLIBRARY) || (srcType == SAVEDPHOTOALBUM)) {
             PluginResult r = new PluginResult(PluginResult.Status.NO_RESULT);
             r.setKeepCallback(true);
             callbackContext.sendPluginResult(r);
-            
+
+            return true;
+        } else if (action.equals("checkForSavedResult")) {
+            if (savedRequestCode > 0 || savedResultCode > 0) {
+                processConfiguration(args);
+                imageUri = Uri.fromFile(createCaptureFile(encodingType));
+                onActivityResult(savedRequestCode, savedResultCode, savedIntent);
+                savedRequestCode = 0;
+                savedResultCode = 0;
+                savedIntent = null;
+            } else {
+                callbackContext.success("");
```

Comment:
I'm not sure it's expected to get an empty string returned on success -- I believe the code will always return the requested result (URI to file, or base64 data). Better to just return an error here?

                callbackContext.error("No saved results available.");